docs/source/examples/elasticsearch_outlet.rst

```diff
@@ -0,0 +1,85 @@
+ElasticSearch Outlet
+--------------------
+
+.. container:: tutorial-block
+
+    In this example we create an implementation of :any:`Outlet` that indexes documents to a running Elasticsearch instance. 
+    
+    **Note**: this example assumes that Elasticsearch is correctly configured and that the index you are indexing documents to exists
+    with the appropriate mappings.  
+
+    #. Extend the :any:`Outlet` with new parameters required when constructing: :code:`es_client` an instance of the elasticsearch python client and :code:`index_name` the name of a pre-existing index in the running cluster.
```

Comment:
Don't hate me for being picky, but I'd capitalise these `E`s and `P`s on proper names.

Also to boost readability I'd add a hyphen after the code blocks, eg. :code:`es_client` - an instance ...